### PR TITLE
webtransport: Update server

### PR DESCRIPTION
**Pull request by bashi**
The example test page (client.html) should work on Chrome 91+ with this update.

### CODE REVIEW

**Comment by bashi** [comment]
@jeffposnick PTAL when you have time?

**Comment by jeffposnick** [comment]
Thank you _very_ much for submitting this! I tried it out with a locally generated key and certificate, and things appeared to work fine with Chrome 93.

However, @vasilvv was the one who wrote the original server code, and I would feel best about letting them weigh in before merging this, just in case there are any other changes that need to be made. I'll reach out to @vasilvv directly via email as well, as I'm not sure if they check their GitHub notifications.

**Comment by jeffposnick** [comment]
Merging after consultation with some members of the team.

File: webtransport/webtransport_server.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+An example WebTransport over HTTP/3 server based on the aioquic library.
+Processes incoming streams and datagrams, and
+replies with the ASCII-encoded length of the data sent in bytes.
+Example use:
+  python3 web_transport_server.py certificate.pem certificate.key
```
Comment (guest271314) [inline webtransport/webtransport_server.py:21]: Should be python3 webtransport_server.py certificate.pem certificate.key

Comment (jeffposnick) [reply webtransport/webtransport_server.py:21]: Fixed in https://github.com/GoogleChrome/samples/commit/3464303a7112f414e755e1df71fe60c9986c2dd6

**Comment by guest271314** [comment]
@bashi Can you kindly update GoogleChrome samples `WebTransport` code concurrent with Chromium source code updates and the updates made to WPT test server? As it stands the current code does not work using aioquic.

**Comment by bashi** [comment]
@yutakahirano FYI

**Comment by yutakahirano** [comment]
<del>Are you talking about https://web.dev/webtransport?</del>

Oops, I was confused, sorry. Yes, we should update https://github.com/GoogleChrome/samples/tree/gh-pages/webtransport.

**Comment by guest271314** [comment]
That too. Basically wherever documentation and codc is exposed. WPT banned me so I cannot contribute there. I have issues open in aioquic repository. 

- https://github.com/aiortc/aioquic/issues/217
- https://github.com/aiortc/aioquic/issues/226

Google Chrome samples uses aioquic, so I expect Chrome authors to be involved in fixing those issues. I gather there are many updates being made. All the more reason to keep the server code here updated. Just do them both at the same time and developers in the field won't be using outdated technology. This same pattern occurred when `QuicTransport` was deprecated; abandoned code was still deployed.

**Pull request merged**